tests/sdk/v2/test_local_client_v2.py

```
@@ -0,0 +1,172 @@
+            result_generator = await client.generate(
+                messages, "qwen3:0.6b", stream=True
+            )
+
+            chunks = []
+            async for chunk_str in result_generator:
+                chunks.append(chunk_str)
+
+            assert len(chunks) == 3
+
+            # Verify that each chunk is a JSON string in the correct format
+            for i, chunk_str in enumerate(chunks):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)